Suggestions native flow (#223)
cowprotocol/docs: docs/cow-protocol/tutorials/cow-swap/native.mdx

It's essential for network operations, like paying transaction fees.

"Wrapped" tokens - for example, Wrapped Ether (WETH) & Wrapped xDAI (wxDAI) - are [ERC20-compatible](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/) versions of these native tokens. 

They're created to improve interoperability with decentralized applications. The native token and its wrapped version are equivalent to each other, and a user is always allowed to turn one into the other ("wrap" or "unwrap") at a rate of `1:1`.

_Buying_ native tokens such as ETH on CoW Protocol is straightforward. However, CoW Protocol is only natively capable of _selling_ ERC20 tokens - not native tokens. 
This is because its intent-based architecture relies on solvers to use the sell token on behalf of its traders.
Doing so is only possible thanks to ERC20's concept of approvals, which allow certain "spender" contracts to manage funds on your behalf.

Despite this, CoW Protocol provides two ways for users to sell their native tokens:

* **Option 1) Wrap native tokens before swapping**: Wrap the native token into an ERC20 token (eg. `ETH` to `WETH`) and then swap WETH gass free on CoW Protocol; or

> * **Option 2) Use the [Eth-flow](/cow-protocol/reference/contracts/periphery/eth-flow)**:  Place a single transaction in a special contract that wraps + places your order.

Reviewer: ```suggestion
1. [Wrap native tokens](#wrap-native-tokens) before swapping**: Wrap the native token into an ERC20 token (eg. `ETH` to `WETH`) and then swap WETH gas-lessly on CoW Protocol; or
2. [Use Eth-flow](#use-eth-flow):  Place a single transaction in a special contract that wraps + places your order.
```

Generally, if wanting to write numbered lists, we do this explicitly :smiley:

Author: For me a number list is less visual, as it can mean sequence in most of the cases

Author: the link is a good idea

> ## Option 1) Wrap native tokens before swapping

Reviewer: ```suggestion
## Wrap native tokens
```

Let's keep the names short, and the numbering is now implicit from the hyperlinks above.

Author: but you are assuming someone reading the whole doc, if they just skim to the titles or use this menu, is not as obvious they need to choose one of these 2:

<img width="1196" alt="image" src="https://github.com/cowprotocol/docs-v2/assets/2352112/6f84c932-e75f-4e97-93ca-9bb55dfc6f98">

Wrapping before swapping requires an extra step. However, users who wrap before swapping get the following benefits: 

> 1. Lower overall fees (if you need to swap multiple times)

Reviewer: ```suggestion
1. Lower overall fees
```
The tip below references the multiple order fee savings so we don't need to duplicate it here

Author: but without this qualification, the statement is just not true! Its not `Lower overall fees`

After wrapping the token, you can simply select the wrapped token as your sell token and continue with your swap.

:::tip

If you plan to sell the native tokens over multiple orders, its best to wrap all it together, so you only pay the gas fee once.

If this is a one-off swap, consider using Eth-flow instead.

> :::

Reviewer: Potentially move this so that it is before the "Wrap native tokens" so it provides guidance when the user sees the list of potential ways to trade native tokens, they can choose which is best for them before drilling down into the specific options.

Author: moved

> ## Option 2) Use Eth-flow

Reviewer: ```suggestion
## Use Eth-flow
```

In an attempt to smooth the user experience, CoW Protocol has introduced the [Eth-flow](/cow-protocol/reference/contracts/periphery/eth-flow) contract. This allows users to automate the [above process](#wrap-native-tokens) and wrap + swap in a single **on-chain** transaction. 

Eth-flow get the following benefits:
1. Lower overall fees (if you need to swap just once)
2. Simpler UX (single transaction)
3. You don't need to approve WETH explicitly in order to sell it

> Using the Eth-flow is faster than wrapping, then swapping. However, because an Eth-flow transactions is on-chain (unlike most CoW Swap swaps), the user must pay an up-front gas fee in the native token. This fee is not refunded if the order fails.

Reviewer: ```suggestion
Using the Eth-flow is faster than wrapping, then swapping. However, because an Eth-flow transactions is on-chain (unlike most CoW Swap swaps), you must pay an up-front gas fee in the native token. This fee is not refunded if the order fails.
```

> ## Option 2) Use Eth-flow {#use-eth-flow}

Author: cool!
